src/test/java/ca/qaguru/oranghrmbatch24/pages/NationalitiesPage.java

```diff
@@ -0,0 +1,170 @@
+package ca.qaguru.oranghrmbatch24.pages;
+import ca.qaguru.oranghrmbatch24.library.PageBase;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.Assert;
+
+import java.time.Duration;
+import java.util.List;
+
+public class NationalitiesPage extends PageBase {
+    private final long WAIT_TIME=5000L;
+    private final long wait_time=20L;
+    private final String NationAddBtn ="//*[@type='button'] [@class='oxd-button oxd-button--medium oxd-button--secondary']";
+    private final String EditNationButton="(//*[@class='oxd-icon-button oxd-table-cell-action-space'])[2]";
+    private final String NationTextBoxField="//div[@class='oxd-form-row']//descendant::input";
+    private final String SaveButton="//button[@type='submit']";
+    private final String CancelButton="//*[@type='button'][@class='oxd-button oxd-button--medium oxd-button--ghost']";
+    private final String DeleteNationButton="(//*[@class='oxd-icon-button oxd-table-cell-action-space'])[1]";
+    private final String DeleteConfirmButton="//*[@class='oxd-button oxd-button--medium oxd-button--label-danger orangehrm-button-margin']";
+    private final String CancelDeleteNationButton="//*[@type='button'][@class='oxd-button oxd-button--medium oxd-button--text orangehrm-button-margin']";
+    private final String LogoutDropdownButton="//*[@class='oxd-userdropdown-name']";
+    private final String LogoutButton="(//*[@role='menuitem'][@class='oxd-userdropdown-link'])[4]";
+    private final String Banner="//*[@class='oxd-toast oxd-toast--success oxd-toast-container--toast']";
+    public NationalitiesPage(WebDriver driver)
+    {
+        super(driver);
+    }
+
+    public void AddButtonVisibleClickable()
```

Comment:
method name has to start with small letter

Reply:
Done